src/sections/chat/ChatSidebarSectionNew.tsx

```diff
@@ -1,12 +1,45 @@
 import { ChatSidebarContentNew } from "components/chat/ChatSidebarContentNew";
 import { ChatSidebarFooterNew } from "components/chat/ChatSidebarFooterNew";
-import React from "react";
+import { ItemVV2 } from "components/reusables/SharedStylingV2";
+import { device } from "config/Globals";
+import { ChatLocalContext } from "contexts/ChatLocalContextNew";
+import React, { useContext } from "react";
+import styled, { useTheme } from "styled-components";
 
 export const ChatSidebarSectionNew = () => {
+
+  const {selectedChat} = useContext(ChatLocalContext);
+
+  const theme = useTheme();
+
   return (
-    <>
+    <Container chatActive={selectedChat}>
```

Comment:
Add type here for container

interface ContainerProps extends ThemeProps<any> {
  chatActive: boolean;
}